packages/ui/src/components/DataStory/clients/ItemsApi.ts

```diff
@@ -1,7 +1,9 @@
+export interface TableItems {
+  atNodeId: string;
+  limit?: number;
+  offset?: number;
+}
+
 export interface ItemsApi {
-  getItems: (options: {
-    atNodeId: string,
-    limit?: number,
-    offset?: number,
-  }) => Promise<any[]>;
-}
\ No newline at end of file
+  getItems: (options: TableItems) => Promise<any[]>;
```

Comment:
We should be able to replace any here with Item or ItemValue or some type like that?